go.mod

```diff
@@ -132,5 +132,3 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
-
-replace github.com/uber/athenadriver => github.com/grafana/athenadriver v0.0.0-20230518203225-a81b0073ac84
```

Comment:
As far as I can tell this fork has zero commits ahead and 23 commits behind upstream, just use upstream.